frontends/bmd/src/pages/review_page.tsx

```diff
@@ -546,11 +548,29 @@ export function ReviewPage(): JSX.Element {
         <Sidebar
           footer={
             <React.Fragment>
-              <SettingsTextSize
-                userSettings={userSettings}
-                setUserSettings={setUserSettings}
-                sidebarWrapper
-              />
+              <ButtonFooterLandscape>
+                {/* <SegmentedButton>
+                  <Button
+                    primary
+                    onPress={() => {}}
+                    aria-label="Change Language"
+                  >
+                    Español
+                  </Button>
+                  <Button onPress={() => {}} aria-label="Change Language">
+                    English
+                  </Button>
+                </SegmentedButton> */}
+                {/* <Button onPress={() => {}} aria-label="Change Language">
+                  English
+                </Button> */}
+                <Button
+                  onPress={() => setUserSettings({ showSettingsModal: true })}
+                  aria-label="Change Settings"
+                >
+                  Settings
+                </Button>
```

Comment:
Is it possible to pull out these settings as a `React.Fragment`?

Reply:
In order to re-use in the landscape and portrait case.

Reply:
Agreed. Done.